### PR TITLE
Fix code example

### PR DESCRIPTION
**Pull request by dingo-d**
A fix after a comment by @GaryJones in https://github.com/WordPress/wpcs-docs/pull/106#discussion_r962182050_

### CODE REVIEW

**Review by GaryJones — APPROVED** [review]

**Review by jrfnl — APPROVED** [review]

**Comment by jrfnl** [comment]
While looking at this line again now... one thing sprang to mind (reason I've not merged this yet):

I believe the `[...]` in the code sample intends to indicate "code here".
However, the square brackets in that bit `[...]` (which I believe come from typical IDEs using them), could also be interpreted as short array/short list brackets.... (by people not using IDEs/using a different IDE).

Considering that that **_always_** comes up as a discussion point again, I wonder if we should change that part of the code sample from `{ [...] };` to `{}`. What do you think ?

(Note: includes removing the redundant semi-colon after the end bracket, which would get flagged by WPCS as an empty statement otherwise)

**Comment by dingo-d** [comment]
I've fixed the code samples as suggested and merged this (it's a small fix). Thanks for the suggestion 👍🏼

File: wordpress-coding-standards/php.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ foo() && bar();
 array( 1, 2, 3 );
 $baz . '-5';
 $term .= 'X';
-if( $object instanceof Post_Type_Interface ) { [...] };
+if ( $object instanceof Post_Type_Interface ) {};
```
Comment (jrfnl) [inline wordpress-coding-standards/php.md:155]: @dingo, thanks for updating this, though this line still has a stray `;` after the closing curly....

Comment (dingo-d) [reply wordpress-coding-standards/php.md:155]: 😱 Doh! Will fix this asap 🙈

**Pull request merged**